crosswalks/bibtex.csv

```diff
@@ -0,0 +1,74 @@
+Property,BibTeX (@softwareversion)
+codeRepository,repository
+programmingLanguage,
+review,
+runtimePlatform,
+targetProduct,
+applicationCategory,
+applicationSubCategory,
+downloadUrl,file
+fileSize,
+installUrl,
+memoryRequirements,
+operatingSystem,
+permissions,
+processorRequirements,
+releaseNotes,
+softwareHelp,
+softwareRequirements,
+softwareVersion,version (1)

```

Comment:
I added a number here to indicate priority of fields, but it might not be consistent with other mappings, so I can remove it.